addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3594,7 +3592,7 @@ var SnippetsMenu = Widget.extend({
             return;
         }
         this.lastElement = srcElement;
-        _.defer(() => {
+        setTimeout(() => {
```

Comment:
same `browser.setTimeout` check your diff to see if there are other occurrences, same for `setInterval`, `clearTimeout`... Have a look at `browser.js`